baseframe/forms/auto.py

```diff
@@ -17,25 +18,25 @@ class ConfirmDeleteForm(Form):
     cancel = SubmitField(__(u"Cancel"))
 
 
-def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False):
+def render_form(form, title, message='', formid='', submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True):
     multipart = False
+    if not formid:
+        formid = 'form-' + buid()
     for field in form:
         if isinstance(field.widget, wtforms.widgets.FileInput):
             multipart = True
-    if form.errors:
-        code = 200  # 400
-    else:
-        code = 200
+    if not with_chrome:
```

Comment:
What if the client wants a chrome-less ajax form? This condition implies that they would be returned `autoform` instead of `ajaxform`?

Reply:
Since this template is just the form without the entire page just like the `ajaxform.html.jinja`. Changed it to use the same.
Fixed